.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   install-dependencies:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:8.10.0
```

Comment:
@adidahiya your linked PR uses 8.9.4, and the build fails with "command not found: yarn"

Reply:
yeah... it looks like the fix did not make it to 8.10 :(

but there is no 8.9.4-browsers... so 🤔